articles/libraries/_includes/_change_get_profile.md

```diff
@@ -1,5 +1,5 @@
 ### Change calls to getProfile()
 
-The deprecated `getProfile()` function was reimplemented in Auth0.js v9. The previous implementation received an [ID Token](/tokens/id-token) as a parameter and returned the user profile. 
+The deprecated `getProfile()` function was reimplemented in Auth0.js v9 as `userInfo()`. The previous implementation received an [ID Token](/tokens/id-token) as a parameter and returned the user profile. 
 
```

Comment:
This is not accurate.

We had a getUserInfo() function in auth0.js v8 that called /userinfo, and a getProfile() one that called /tokeninfo. In v9 we made getProfile() call /userinfo.